nx/lib/nx/binary_backend.ex

```diff
@@ -783,7 +782,7 @@ defmodule Nx.BinaryBackend do
   def floor(out, tensor), do: element_wise_unary_op(out, tensor, &:erlang.floor/1)
```

Comment:
What about floor and round? :)

Reply:
Missed these ones :)
I'll take a look into torch/numpy to see if they support this

Reply:
Torch doesn't support any of floor, round and ceil. Numpy supports round, but neither floor nor ceil. How should we go about this?

Reply:
Numpy's round behaves the same as Kernel.round/1

Reply:
Oh, I see. Let's not support it for now then.